lib/preview/defaults.js

```diff
@@ -20,11 +20,24 @@ module.exports = function defaults (options) {
   const quality = options.quality || null
   const limit = options.limit || 19008
   const tileSize = options.tileSize || 256
+  const concurrency = options.concurrency || 32
```

Comment:
Can we set this as `poolMaxWaitingClients` by default if no value is passed instead?

Reply:
The only point in common between `renderer` & `preview` is `getTile` method and there is no way to get that value from `getTile` function... :(

Reply:
BTW, I will pass that value in windshaft.

Reply:
There I have both, the renderer and preview.

Reply:
:ok_hand: Then I'd leave a comment there (or in the README) to note that this value is related to poolMaxWaitingClients (something like `//to match poolMaxWaitingClients`).